index.html

```diff
@@ -1,9 +1,32 @@
 <!DOCTYPE html>
-<head>
-    <title>Mango Site</title>
-</head>
-<body>
-    <!-- Comment for the body section -->
-    <h1>Mango Site</h1>
-    <p>This is the first version of the mango site</p>
-</body>
\ No newline at end of file
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Mango Site</title>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+    <body>
+        <div class="title">
+            <h1>PENNY JUICE</h1>
+            <h3>IT MAKES CENTS</h3>
+        </div>
+        <div class="nav-bar">
+            <ul>
+                <li><a href="Home">Home</a></li>
+                <li><a href="ChildCarejuice">ChildCare Juice</a></li>
+                <li><a href="Order Now">Order Now</a></li>
```

Comment:
```suggestion
                <li><a href="#">Order Now</a></li>
```
For all the elements in the li

Reply:
separated with different div elements and alignment has changed and buttons with box shadow